jwst/resample/resample_spec.py

```diff
@@ -651,18 +651,23 @@ def _spherical_sep(j, k, wcs, xyz_ref):
     return 1 - np.dot(_S2C(ra, dec), xyz_ref)
 
 
-def _find_nirspec_output_sampling_wavelengths(wcs_list, targ_ra, targ_dec, fast=True):
+def _find_nirspec_output_sampling_wavelengths(wcs_list, targ_ra, targ_dec, mode='median'):
+    assert mode in ['median', 'fast', 'accurate']
```

Comment:
I see the `accurate` option for `mode` listed here, but I don't see any use of that option in the code. So what happens if someone passes in a value of `accurate`?

Reply:
if that happens, the statement `fast=mode == 'fast'` on L669 will evaluate to `False`